cpp/tests/utilities/test_graphs.hpp

```diff
@@ -163,11 +165,121 @@ class Rmat_Usecase : public detail::TranslateGraph_Usecase {
              bool>
   construct_edgelist(raft::handle_t const& handle, bool test_weighted) const
   {
-    // TODO: Tease through generate_graph_from_rmat_params
-    //       to extract the edgelist part
-    // Call cugraph::translate_vertex_ids(handle, d_src_v, d_dst_v, base_vertex_id_);
+    constexpr bool symmetric{true};
```

Comment:
Do we always symmetrize R-mat graphs? This is necessary for WCC or Louvain, but there are many other algorithms that work with asymmetric R-mat graphs.

Reply:
> Do we always symmetrize R-mat graphs? This is necessary for WCC or Louvain, but there are many other algorithms that work with asymmetric R-mat graphs.

@ChuckHastings I'm waiting for your answer for this to approve this PR.

Reply:
That was the intention of my FIXME comment.  Right now the symmetrization flag is inside the test graph classes.  I think the goal would be to set the symmetrization flag to false for *all* of the test graph classes (except file which would read it from the MM file and return the intended value) and add a test graph wrapper that will symmetrize any test graph so that Louvain and WCC could use that wrapper around rmat and other generators.

Reply:
Where is the FIXME comment?

```Rmat_Usecase``` has private member variable ```  bool undirected_{};```

So, this should be used instead of redefining ```symmetric``` here and ignoring the member variable. I assume users will be surprised if they set Rmat_Usecase.undirected_ = false and get symmetric graphs.

And it seems like ```construct_graph``` is still calling
```
return generate_graph_from_rmat_params<vertex_t, edge_t, weight_t, store_transposed, multi_gpu>(
      handle,
      scale_,
      edge_factor_,
      a_,
      b_,
      c_,
      seed_,
      undirected_,
      scramble_vertex_ids_,
      test_weighted,
      renumber,
      partition_ids,
      comm_size);
````
So, ```construct_edgelist``` here is actually unused. Are you planning to replace ```generate_graph_from_rmat_params``` but haven't done the job, yet? Why not replace in this PR if it's the intention?

Reply:
The original implementation work here was truncated and a partially implemented capability was pushed many months ago.  This is unfinished tech debt.  I will look into making that change as part of this PR.